Libplanet/Store/TrieStateStore.cs

```diff
@@ -29,11 +30,16 @@ public class TrieStateStore : IStateStore, IDisposable
         /// <see cref="MerkleTrie"/> in internal.</param>
         /// <param name="stateHashKeyValueStore">The storage to store state hash corresponding to
         /// block hash.</param>
+        /// <param name="secure">A flag whether to use <see cref="MerkleTrie"/> as secure mode.
+        /// <seealso cref="MerkleTrie(IKeyValueStore, INode?, bool)"/>.</param>
         public TrieStateStore(
-            IKeyValueStore stateKeyValueStore, IKeyValueStore stateHashKeyValueStore)
+            IKeyValueStore stateKeyValueStore,
+            IKeyValueStore stateHashKeyValueStore,
+            bool secure = false)
```

Comment:
We don't need edit changelog for this? It is public class and public constructor

Reply:
Because `TrieStateStore` has not released yet since it introduced.